### PR TITLE
Add sb tag and fix typo

### PR DESCRIPTION
**Pull request by Rodrun**
Add smash bros tag

### CODE REVIEW

File: serverstf/tags/mode.py [hunk 1]
```diff
@@ -46,6 +46,12 @@ def rd(info, players, rules, tags):
...
 
-@tag("mode:medival", ["tf2"])
+@tag("mode:medieval", ["tf2"])
 def medieval(info, players, rules, tags):
     return "tf2" in tags and rules["rules"].get("tf_medieval") == "1"
+
+@tag("mode:sb", ["tf2", "mode:arena"])
+def sb(info, players, rules, tags):
+    return ("tf2" in tags
+            and "mode:cp" in tags
```
Comment (Holiverh) [inline serverstf/tags/mode.py]: This condition can never evaluate to `True`. Is it meant to be `mode:arena` per change?

File: serverstf/tags/mode.py [hunk 2]
```diff
@@ -46,6 +46,12 @@ def rd(info, players, rules, tags):
     return "tf2" in tags and rules["rules"].get("tf_gamemode_rd") == "1"
 
 
-@tag("mode:medival", ["tf2"])
+@tag("mode:medieval", ["tf2"])
 def medieval(info, players, rules, tags):
     return "tf2" in tags and rules["rules"].get("tf_medieval") == "1"
+
+@tag("mode:sb", ["tf2", "mode:arena"])
```
Comment (Holiverh) [inline serverstf/tags/mode.py]: I'm reluctant to let mods occupy the `mode:` namespace but it'll do for now.

File: serverstf/tags/mode.py [hunk 3]
```diff
@@ -46,6 +46,12 @@ def rd(info, players, rules, tags):
...
+def sb(info, players, rules, tags):
```
Comment (Holiverh) [inline serverstf/tags/mode.py]: Needs a better name and a docstring.

File: serverstf/tags/mode.py [hunk 4]
```diff
@@ -46,6 +46,12 @@ def rd(info, players, rules, tags):
     return "tf2" in tags and rules["rules"].get("tf_gamemode_rd") == "1"
 
 
-@tag("mode:medival", ["tf2"])
+@tag("mode:medieval", ["tf2"])
 def medieval(info, players, rules, tags):
     return "tf2" in tags and rules["rules"].get("tf_medieval") == "1"
+
```
Comment (Holiverh) [inline serverstf/tags/mode.py]: There should be two blank lines here.

Comment (Rodrun) [inline serverstf/tags/mode.py]: Okay

**Comment by Rodrun** [comment]
:+1:

**Pull request merged**